alica_engine/include/engine/BasicBehaviour.h

```diff
@@ -61,7 +62,7 @@ class BasicBehaviour : private RunnableObject
     const VariableGrp& getVariables() const { return _behaviour->getVariables(); }
     const Variable* getVariable(const std::string& name) const { return _behaviour->getVariable(name); };
 
-    void terminate() { stop(); };
+    void terminate(RunningPlan* rp) { stop(rp); };
```

Comment:
What is the point of this function?  Why not call stop directly?

Reply:
I'm not sure myself. I just left the naming/pattern as it was.